SingularityBase/src/main/java/com/hubspot/mesos/MesosUtils.java

```diff
@@ -309,4 +311,11 @@ private static Ranges subtractRanges(Ranges ranges, Ranges toSubtract) {
     return remaining;
   }
 
+  public static Path getTaskDirectoryPath(String taskId) {
+    return Paths.get(getSafeTaskIdForDirectory(taskId)).toAbsolutePath();
```

Comment:
Not sure if we can get away with this implementation as a static method since it's depending on the current directory here -- might make more sense to have this take 2 args (the base path and then the taskId)